metadata.json

```diff
@@ -69,5 +69,5 @@
   "description": "Tasks that manipulate a package",
   "pdk-version": "1.10.0",
```

Comment:
Nothing to change, but remember to update your pdk to 1.11

Reply:
updated pdk version